packages/cloud-cognitive/src/index-without-carbon.scss

```diff
@@ -5,25 +5,9 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import './global/styles/imported-carbon-modules';
-
+@import 'common-settings/scss/imported-carbon-modules';
```

Comment:
I believe this may break in a production Sass environments because the `common-settings` package is currently both private and listed in `devDependencies`.

Reply:
Ah, good point. I may have headed down the wrong route here...